src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/Mocks/IVsHiearchyItemFactory.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.Shell;
+using Moq;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS
+{
+    internal static class IVsHierarchyItemFactory
+    {
+        public static IVsHierarchyItem Create()
+        {
+            return Mock.Of<IVsHierarchyItem>();
+        }
+
+        public static IVsHierarchyItem Implement(string text = null,
```

Comment:
Could you say what method/getter you're implementing, in case we have to add new things in the future?